src/contexts/mobile-menu.tsx

```diff
@@ -19,7 +19,7 @@ import { useNoScrollBody } from 'hooks/use-no-scroll-body'
 /**
  * Should correspond to --dev-dot-hide-mobile-menu
  */
-const DEFAULT_NAV_HEADER_DESKTOP_WIDTH = 924
+const DEFAULT_NAV_HEADER_DESKTOP_WIDTH = 1230
```

Comment:
just curious about how you landed on `1230` here

Reply:
Deferring to @LeahMarieBush as I think she had made this adjustment :+1:

Reply:
@heatlikeheatwave This got changed because the new product panel is wider than the previous. 1230 is around when the panel gets to the edge of the screen. Without this, the product panel would've pushed off the side of the page and caused white space on the right side

Reply:
That makes sense. Did you try changing the position of the panel, so it is more centered in the screen? Like so 👇 

<img width="1411" alt="Screenshot 2024-09-05 at 14 54 01" src="https://github.com/user-attachments/assets/53f54aab-2e75-4931-b0bb-c46f98ac9ed7">

Reply:
@heatlikeheatwave Ah, so the panel would drop down as if it was aligned to the logo, instead of to the "Products" trigger in the nav? I think that works, would have to test it out of course!

However, is there a plan to have more dropdowns in the nav in the future? If there is, I would assume they would drop down aligned to their triggers, which might make this product dropdown look out-of-place by comparison 🤔

Reply:
> Ah, so the panel would drop down as if it was aligned to the logo, instead of to the "Products" trigger in the nav?

Yeah, exactly! That way we don't merge in a change that impacts the layout of other pages where this menu does not exist.

> However, is there a plan to have more dropdowns in the nav in the future? 

Not that I currently know of. The nav is different for product documentation & tutorials, and seems like it will be for the foreseeable future (unless there's another dev portal nav project in the works?).

I think it makes sense to adjust the position of the nav to fit within the existing breakpoints as that's our current use case, rather than making a change that impacts the layout of _all_ pages, but could accommodate a future and unknown nav change. Does that make sense?

Reply:
This makes sense @heatlikeheatwave ! This approach will still leave the nav looking kinda cut off between widths `924px` and `1013px` though by my calculations (`1013px` = `989px` dropdown width on desktop + `24px` to align the panel as it is in your screensheet, and `924px` is when the panel would change layout to mobile on developer). 

Given the small window of viewports this cutoff will appear in, that seems like a fair trade to me to avoid affecting the layout of other pages while minimizing visual issues. Is this reasonable?

Reply:
@heatlikeheatwave It does make sense :+1: I've implemented this change locally and it does the trick, but I'm getting feedback from design on how this looks before I commit anything.

Reply:
@heatlikeheatwave Update: design is onboard with moving the panel over! Ben suggested also reducing nav item padding, but I will see how the panel behaves with just moving it over first, as that padding change will affect both usages of the panel.

Reply:
Awesome! Thanks for following up with design about it. I appreciate all the extra work you've put into this feature (:

Reply:
Thanks @heatlikeheatwave 🎉 

I've implemented the move-over in this commit:
e8809fb9e7369a05635c23cace9d53795fccd2de

Changing the padding of items did nothing to the width of the dropdown, I'm assuming because of the use of CSS grid for layout. I don't think the `16px` of saved space is a blocker for launch, personally, but it is something to see if we can address in a follow-up if we don't get to it 👍